<launch>
   <param name="steering_increment" value="0.5" />
   <param name="throttle_increment" value="2.5" />
   <param name="brake_increment" value="9" />
   <param name="can_comm_port" value="/dev/ttyUSB0"/>
   <param name="can_steer_id" value="1"/>
   <param name="can_brake_id" value="2"/>
   <param name="throttle_comm_port" value="/dev/talos/" />
   <group ns="robo_cart">
      <node pkg="drive_train" type="drive_train_node" name="cart_base"/>
   </group>
</launch>
